### PR TITLE
Only update published dates on publish and return the page data

### PR DESCRIPTION
**Pull request by tjmkruger**

### CODE REVIEW

File: src/modules/page/publishPage.js [hunk 1]
```diff
@@ -10,13 +10,17 @@ module.exports = {
...
     page.published = true;
+    page.lastPublished = new Date();
+
+    let publishedPage = page;
 
     return $startTransaction(async session => {
       const savedPage = await $save(collection, page, { session });
-      const publishedPage = await $publish(collection, savedPage, { session });
+      publishedPage = await $publish(collection, savedPage, { session });
+      console.log('🚀 ~ file linked: publishPage.js ~ line 19 ~ exec ~ publishedPage', publishedPage);
```
Comment (BGillyVII) [inline src/modules/page/publishPage.js:20]: Log not required

File: src/modules/page/publishPage.js [hunk 2]
```diff
@@ -10,13 +10,17 @@ module.exports = {
...
       await $record(collection, 'publish', { data: publishedPage, user }, { session });
       if ($publishing.onPublish) await $publishing.onPublish(page);
-      return publishedPage;
-    });
+      // });
```
Comment (BGillyVII) [inline src/modules/page/publishPage.js:23]: not needed

File: src/modules/page/publishPage.js (hunk 1 above)
Comment (tjmkruger) [reply src/modules/page/publishPage.js]: removed

File: src/modules/page/publishPage.js (hunk 2 above)
Comment (tjmkruger) [reply src/modules/page/publishPage.js]: removed

**Pull request merged**